cyences_app_for_splunk/bin/upgrade_steps.py

```diff
@@ -33,11 +33,29 @@ def upgrade_4_0_0(session_key, logger):
     response = service.jobs.oneshot(SEARCH_QUERY, output_mode="json", earliest_time='-4w@w', latest_time='@w')
     handle_results(response, logger)
 
+
+def upgrade_4_3_0(session_key, logger):
+    DEVICE_INVENTORY_SEARCH_QUERY = '| savedsearch "Device Inventory Backfill - V2"'
+
+    service = client.connect(token=session_key, app=cs_utils.APP_NAME)
+
+    logger.info("Running Device Inventory Backfill search for last 7 day timerange")
+    response = service.jobs.oneshot(DEVICE_INVENTORY_SEARCH_QUERY, output_mode="json", earliest_time='-7d@m', latest_time='now')
+    handle_results(response, logger)
+    time.sleep(60)
+
+    USER_INVENTORY_SEARCH_QUERY = '| savedsearch "User Inventory - Lookup Backfill"'
```

Comment:
@hardikhdholariya - Please check the Splunk logs to see if this search is not getting timeouted. And check how much time it is taking to run.

Reply:
yes, verified that it's not getting timeouted. On cerberos instance, entire search tooks approx. 66 min. where device inventory completes in 6-7 min and user inventory took approx. 1 hour.

FYI, I reduced the time range here compared to backfill searches time range.
device inventory backfill runs for last 30 days and here we ran it for last 7 days.
user inventory backfill runs for last 7 days and here we ran it for last 1 day.